orchagent/copporch.cpp

```diff
@@ -75,12 +77,68 @@ std::map<string, sai_packet_action_t> packet_action_map = {
     {"transit", SAI_PACKET_ACTION_TRANSIT}
 };
 
+std::vector<sai_hostif_trap_id_t> default_trap_ids = {
+    SAI_HOSTIF_TRAP_ID_TTL_ERROR
+};
+
+const string default_trap_group = "default";
+
 CoppOrch::CoppOrch(DBConnector *db, string tableName) :
     Orch(db, tableName)
 {
     SWSS_LOG_ENTER();
+    initDefaultTrapGroup();
+    initDefaultTrapIds(default_trap_ids);
 };
 
+void CoppOrch::initDefaultTrapIds(std::vector<sai_hostif_trap_id_t> &trap_ids)
+{
+
+    sai_attribute_t attr;
+    std::vector<sai_attribute_t> trap_id_attrs;
+
+    attr.id = SAI_HOSTIF_TRAP_ATTR_PACKET_ACTION;
+    attr.value.s32 = SAI_PACKET_ACTION_TRAP;
+    trap_id_attrs.push_back(attr);
+
+    attr.id = SAI_HOSTIF_TRAP_ATTR_TRAP_GROUP;
+    attr.value.oid = m_trap_group_map[default_trap_group];
+    trap_id_attrs.push_back(attr);
+
+    attr.id = SAI_HOSTIF_TRAP_ATTR_TRAP_CHANNEL;
+    attr.value.s32 = SAI_HOSTIF_TRAP_CHANNEL_NETDEV;
+    trap_id_attrs.push_back(attr);
+
+    if (!applyAttributesToTrapIds(default_trap_ids, trap_id_attrs))
+    {
+        string err_msg = "Failed applying default trap Ids.";
+        SWSS_LOG_ERROR("%s", err_msg.c_str());
+        throw runtime_error(err_msg);
```

Comment:
map throws exception when invalid trap_id is specified for example, so in some cases the exceptions will be caught.
I can remove the exceptions for which sairedis will always return success